runner/config_schema.yaml

```diff
@@ -42,80 +42,102 @@ definitions:
             type: string
           relative_to:
             $ref: '#/definitions/path'
+  plugin_group:
+    oneOf:
+      - type: array
+        default: []
+        items:
+          type: object
+          additionalProperties: false
+          properties:
+            name:
+              description: An optional name for the plugin
+              default: ""
+              type: string
+            path:
+              $ref: '#/definitions/path'
+              description: Path to the directory containing the SO.
+            config:
+              default: {}
+              type: object
+              additionalProperties:
+                type: string
+              description: Vars that get passed to the build system
+          required:
+            - path
+  flow:
+    oneOf:
+      - type: array
+        default: []
+        items:
+          type: object
+          additionalProperties: false
+          properties:
+            plugin_group:
+              $ref: '#/definitions/plugin_group'
 type: object
 additionalProperties: false
 properties:
-  plugin_groups:
+  flows:
     type: array
     default: []
+    description: (Plugin) Flows to launch in runner (flows run independently)
     items:
       type: object
       additionalProperties: false
       properties:
-        plugin_group:
-          type: array
-          default: []
-          items:
-            type: object
-            additionalProperties: false
-            properties:
-              name:
-                description: An optional name for the plugin
-                default: ""
-                type: string
-              path:
-                $ref: '#/definitions/path'
-                description: Path to the directory containing the SO.
-              config:
-                default: {}
-                type: object
-                additionalProperties:
-                  type: string
-                description: Vars that get passed to the build system
-            required:
-              - path
+        flow:
+          $ref: '#/definitions/flow'
+  append:
+    type: object
+    default: { plugin_group: [] }
+    additionalProperties: false
+    description: Configuration group to append to all included flows
```

Comment:
Just to make sure I understand this correctly, this really does mean _all_? So if you had four flows and two appended plugins, the two plugins will get appended to each flow?

Reply:
Yes, both plugins with be appended to each flow. Would "each" be better?

Reply:
No, the writing is fine, I was just thinking that given that it's _all_, we could theoretically list all the common plugins here, and only list the flow-specific ones under `flows`. That would have been nice, but then I remembered that, among other things, plugin order matters.

Reply:
Right that make sense. I was thinking about also adding a similar `prepend` option, but that only alleviates the problem slightly.